x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -136,17 +136,18 @@ export const calculateVersion = (
   // the version number if only the enabled/disabled flag is being set. Likewise if we get other
   // properties we are not expecting such as updatedAt we do not to cause a version number bump
   // on that either.
-  const removedNullValues = pickBy<UpdateProperties>(
-    (value: unknown) => value != null,
-    updateProperties
-  );
+  const removedNullValues = removeUndefined(updateProperties);
   if (isEmpty(removedNullValues)) {
     return currentVersion;
   } else {
     return currentVersion + 1;
   }
 };
 
+export const removeUndefined = (obj: object) => {
+  return pickBy((value: unknown) => value != null, obj);
+};
```

Comment:
I think the technical term would be `removeNil` as this would be removed null and undefined values. The original was `removedNullValues` but that wasn't accurate as well.

At least nil is the terminology that lodash uses:
https://lodash.com/docs/4.17.15#isNil

No changes asked, just pointing it out.